src/blocks/block-layout/styles/style.scss

```diff
@@ -107,3 +107,14 @@
 		}
 	}
  }
+
+/**
+ * Twenty Twenty theme styles
+ * Let's look at loading these conditionally if the theme is active
+ */
```

Comment:
@mikemcalister are you looking to add more styles for this theme before we merge or is this it for now?

Reply:
This is it for now. We can make a dedicated sweep through the theme once the styles are solidified since they are still changing.